build.sbt

```diff
@@ -371,7 +371,8 @@ lazy val docs = project
     libraryDependencies ++= Seq(
       "com.typesafe.play" %% "play-jdbc"   % playVer.value % Test,
       "com.typesafe.play" %% "play-specs2" % playVer.value % Test,
-      "com.h2database"     % "h2"          % "1.4.199"
+      "com.google.inject" % "guice" % "5.1.0" % Test, // TODO: Remove when depending on play-jdbc/-specs2 2.8.x+
```

Comment:
`playVer` is 2.6.14.... So need to pull in newer Guice version that works with Java 17... It's only in test scope however. But I think this needs more refactoring...